Samples/1c-multiple-input-swaggers/Client/IStorageManagementClient.cs

```diff
@@ -0,0 +1,63 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
```

Comment:
why is this part of this PR? I'd expect zero change in code generation due to your changes